report.c

```diff
@@ -378,48 +380,47 @@ void report_counters()
   printPgmString(PSTR("}\r\n"));
 }
 
-//Prints voltage data: motor volts.
+// Prints voltage data for Motors (C,X,Y,Z) and Force sensor.
 void report_voltage()
 {
+  uint8_t i;
+  calculate_motor_voltage();
+  printPgmString(PSTR("|"));
+  for (i = 0; i<VOLTAGE_SENSOR_COUNT; i++){
```

Comment:
I am nervous about reading all 5 sensors on every tick.  There seems to be no reason to do this.  I think it's better to read one at a time, per tick.

Either: 
x->y->z->c->F->repeat
or
x->F->y->F->z->F->c->F->repeat